client/src/Components/Card/Card.jsx

```diff
@@ -3,31 +3,31 @@ import Spinner from '../Spinner/Spinner';
 import * as Style from './Card.styles';
 
 const Currentweather = ({ data }) => {
-  const weatherData = data[0];
+  const [weatherData] = data;
+  console.log(weatherData);
```

Comment:
please don't forget to remove console logs

Reply:
Done. Will make another pass for unneeded console.log in further PRs